__tests__/app.test.js

```diff
@@ -90,6 +90,40 @@ describe('GET /api/articles:article_id', () => {
 //     });
 // });
 
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('STATUS 200: reponds with an array of comments for the given article_id', () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(result => {
+            const {comments} = result.body
+            console.log(comments)
+            expect(comments.length).toBe(11)
+            for (const comment of comments) expect(comment.article_id).toBe(1)
+            expect(comments).toBeSortedBy("created_at", {descending: true})
```

Comment:
This is nice assertion but I think if you included my suggestion above in this test (checking the shape of the data) this could put into it's own test.